javatests/arcs/showcase/instant/Calendar.kt

```diff
@@ -0,0 +1,40 @@
+@file:Suppress("EXPERIMENTAL_FEATURE_WARNING", "EXPERIMENTAL_API_USAGE")
+
+package arcs.showcase.instant
+
+import arcs.jvm.host.TargetHost
+import arcs.sdk.ArcsDuration
+import arcs.sdk.ArcsInstant
+import arcs.showcase.ShowcaseHost
+
+// LocalDate date = LocalDate.parse("9999-12-31");
+// ArcsInstant instant = date.atStartOfDay(ZoneId.of("Europe/Paris")).toInstant();
+
+typealias Event = AbstractCalendar.Event
+
+@TargetHost(ShowcaseHost::class)
+class Calendar : AbstractCalendar() {
+    override fun onReady() {
+        handles.events.store(
```

Comment:
You could use storeAll() and store all 3 events together.

Reply:
I'd have to construct a collection to do that and there are no other showcases doing that at the moment.

If possible I'll just demo Instant, not introduce other features.